tests/reference-implementation/processor.js

```diff
@@ -23,7 +23,7 @@ function getReferenceKey(nodes, resource) {
 }
 
 
-export async function* processResources(resourceGenerator, configIn) {
+export async function* processResources(resourceGenerator, configIn, options) {
```

Comment:
```suggestion
export async function* processResources(resourceGenerator, configIn, options = {
  "enforceFhirpathSubset": false }) {
```

Does it work to document this structure explicitly inline with defaults? 

Also for whatever reason the name "validatePathToSubset" isn't totally self explanatory to me.

Reply:
Sure, I'm fine renaming the option and will update to be explicit about defaulting to false.